### DISCUSSION TITLE
Menu allows you to insure high-end cars

### DISCUSSION THREAD

**Discussion by ItsEntDev** [YimMenu/YimMenu#2788] [Q&A] [answered]
YimMenu seems to passively allow you to take high-end cars into Benny's and LSC. You can also insure them. This doesn't seem to be Forge Model or even an actual toggle? I've been looking through the source but I don't see anything. Which bit of code is doing this? I'd like to know so I can use Lua to replicate it to other menus.

**Comment by inxanedev** [comment, marked as answer]
https://github.com/YimMenu/YimMenu/pull/2524

The "forge model for LSC" technique is for external menus I believe, internal menus don't need that because you can just make LSC accept any car. Buying insurance for vehicles is possible for high end cars even without mod menus, but you just can't drive them into LSC in the first place

**Reply by ItsEntDev** [reply]
Ah, thank you :) Forge Model is also available in internal menus as well.
